mama/c_cpp/src/c/clientmanageresponder.c

```diff
@@ -90,231 +90,234 @@ static void commandDestroyCB (wList list, void *element, void *closure)
+            while((ret == MAMA_STATUS_OK) && (nextTransportIndex < impl->numberTransportBridges))
+            {
+                /* The publisher allows commands to respond to point to point requests
+                 * without creating a new publisher, this must be created using the correct
+                 * transport bridge.
+                 */
+                ret = mamaPublisher_createByIndex(
+                    nextPublisher,
+                    impl->mTransport,
+                    nextTransportIndex,
+                    NULL,
+                    NULL,
+                    MAMA_CM_PUB_TOPIC,
+                    NULL,
+                    NULL,
+                    NULL);
+                if(ret == MAMA_STATUS_OK)
+                {
+                    /* Allocate the subscription */
+                    ret = mamaSubscription_allocate(nextSubscription);
+                    if(ret == MAMA_STATUS_OK)
+                    {
+                        /* Set the subscription's transport index to ensure that the right transport is used. */
+                        mamaSubscription_setTransportIndex(*nextSubscription, nextTransportIndex);
+                  
+                        /* Create the subscription */
+                        ret = mamaSubscription_createBasic(
+                            *nextSubscription, 
+                            impl->mTransport, 
+                            internalQueue, 
+                            &callbacks, 
+                            MAMA_CM_TOPIC, 
+                            impl);
                         if(ret == MAMA_STATUS_OK)
                         {
                             /* We don't want the CM subs to show up on the stats logger */
                             mamaSubscription_setLogStats(*nextSubscription, 0);
                         }
 
-					}
-				}
+                    }
+                }
 
-				/* Increment the counts for the next iteration */
-				nextTransportIndex ++;
-				nextPublisher ++;
-				nextSubscription ++;
-			}			
-		}    
-	}
+                /* Increment the counts for the next iteration */
+                nextTransportIndex ++;
+                nextPublisher ++;
+                nextSubscription ++;
+            }            
+        }    
+    }
 
-	return ret;
+    return ret;
 }
 
-/*	Description	:	This function will create the cm responder, the responder will listen for management messages from
- *					the currently active feed handlers. This is done by creating a publisher and a subscription for each
- *					sub transport bridge in the main transport.
- *	Arguments	:	responder				[O] To return the new responder.
- *					tport					[I] The main transport.
- *					numberTransportBridges	[I] The number of transport bridges.
- *	Returns		:	MAMA_STATUS_NOMEM
- *					MAMA_STATUS_OK
+/*    Description    :    This function will create the cm responder, the responder will listen for management messages from
```

Comment:
Again, can you revert all whitespace corrections between here and the end of the file please?